x-pack/plugins/security_solution/public/common/hooks/use_url_state.ts

```diff
@@ -29,7 +29,8 @@ export const useUrlState = () => {
 
 export const URL_PARAM_KEY = {
   appQuery: 'query',
-  eventFlyout: 'eventFlyout',
+  eventFlyout: 'eventFlyout', // TODO remove when we assume it's been long enough that all users should use the newer `flyout` key
```

Comment:
We could also mark this key as `@deprectated`.

I was wondering if it made sense to set up some sort of tracking dashboard for how often we see `eventFlyout` vs `flyout`. Then, once we see `eventFlyout` drops to less than one percent, we could drop it altogether.

Reply:
great idea, I'll add the `@deprecated` and will investigate if there is something that we have in place today that would allow us to get that info!

Reply:
done!